common/src/com/thoughtworks/go/remote/work/ConsoleOutputTransmitter.java

```diff
@@ -32,6 +36,7 @@
     private CircularFifoBuffer buffer = new CircularFifoBuffer(10 * 1024); // maximum 10k lines
     private Integer sleepInSeconds;
     private final ConsoleAppender consoleAppender;
+    private final SimpleDateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
```

Comment:
I think it should just be:

```
HH:mm:ss line
```

The date is usually redundant. So, instead of:

```
[2015-02-09 07:52:31][go] setting environment variable ...
```

it should be:

```
07:52:31 [go] setting environment variable ...
```

or *maybe* the one below (though I prefer the one above):

```
[07:52:31][go] setting environment variable ...
```